src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__unit stars--0">
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+    </div>
+    <div class="stars__unit stars--1">
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+      <div class="star__unit"></div>
+    </div>
+    <div class="stars__unit stars--2">
```

Comment:
```suggestion
    <div class="stars stars--2">
```

just stars

Reply:
Okay, I'll do it (:

Reply:
By the way, could I implement this via `ul>li` instead of `div>div` ?)